src/libs/API/parameters/UpdatePolicyAddressParams.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: Change API endpoint parameters format to make it possible to follow naming-convention
```

Comment:
Has an issue been created for it?

Reply:
cc @danieldoglas

Reply:
@deetergp is working on it here: https://github.com/Expensify/Web-Expensify/pull/41840